plugins/confluence.go

```diff
@@ -118,6 +123,26 @@ func (p *ConfluencePlugin) getTotalSpaces() ([]ConfluenceSpaceResult, error) {
 	return totalSpaces.Results, nil
 }
 
+func (p *ConfluencePlugin) getSpecificSpaces() (*ConfluenceSpaceResponse, error) {
+	url := fmt.Sprintf("%s/rest/api/space?", p.URL)
+	for _, space := range p.Spaces {
+		url += fmt.Sprintf("spaceKey=%s&", space)
+	}
```

Comment:
add this to `getSpaces` implementation instead of creating a duplicate function `getSpecificSpaces()`